WcaOnRails/public/404.html

```diff
@@ -1,67 +1,119 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>The page you were looking for doesn't exist (404)</title>
-  <meta name="viewport" content="width=device-width,initial-scale=1">
-  <style>
-  body {
-    background-color: #EFEFEF;
-    color: #2E2F30;
-    text-align: center;
-    font-family: arial, sans-serif;
-    margin: 0;
-  }
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<link href="http://fonts.google.apis.com/css?family=" rel="stylesheet" type="text/css" />
+		<link href="/assets/wca-0e38e06e4765b5d1c9c349f9660ae864f509c331e281c1e617e8542cf32a48b3.ico" rel="icon" type="image/x-icon" />
+		<title>
+			Error 404
+		</title>
+		<style type="text/css">
+			/* apply to all */
+			* {
+				color       : #EEE ;
+				height      : 100% ;
+				font-family : 'Droid Sans' , sans-serif ;
+				text-align  : center ;
 
-  div.dialog {
-    width: 95%;
-    max-width: 33em;
-    margin: 4em auto 0;
-  }
+				/* text selection */
+				-khtml-user-select   : none ;
+				-moz-user-select     : none ;
+				-o-user-select       : none ;
+				-ms-user-select      : none ;
+				-webkit-touch-select : none ;
+				-webkit-user-select  : none ;
+			}
 
-  div.dialog > div {
-    border: 1px solid #CCC;
-    border-right-color: #999;
-    border-left-color: #999;
-    border-bottom-color: #BBB;
-    border-top: #B00100 solid 4px;
-    border-top-left-radius: 9px;
-    border-top-right-radius: 9px;
-    background-color: white;
-    padding: 7px 12% 0;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
+			/* equivalent heights for all 3 subbody tags */
+			header , footer , main { height : 33% }
 
-  h1 {
-    font-size: 100%;
-    color: #730E15;
-    line-height: 1.5em;
-  }
+			/* full body rules */
+			html {
+				background :
+					linear-gradient(
+					rgba( 0 , 0 , 0 , 0.75 ) ,
+					rgba( 0 , 0 , 0, 0.75 ) ) ,
+					url( '/images/404_cube.jpg' ) ;
 
```

Comment:
Yeah I originally had it as 404_cube because it was a 404 page that I made first. I realized I linked it in all the error pages I updated to changed the filename but must have forgot to change the URL referenced by the CSS in the files.